installing/installing_aws/ipi/installing-aws-multiarch-support.adoc

```diff
@@ -0,0 +1,70 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ipi-aws-multiarch-support"]
+= Installing an AWS cluster with the support for configuring multi-architecture compute machines
+include::_attributes/common-attributes.adoc[]
+:context: ipi-aws-multiarch-support
+
+toc::[]
+
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures.
+
+[NOTE]
+====
+When there are nodes with multiple architectures in your cluster, the architecture of your image must be consistent with the architecture of the node. You need to ensure that the pod is assigned to the node with the appropriate architecture and that it matches the image architecture. For more information on assigning pods to nodes, see link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assigning pods to nodes].
+====
```

Comment:
Can we add the same recommendation as in the other pages?

https://docs.openshift.com/container-platform/4.16/post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-gcp.html#multi-architecture-modify-machine-set-gcp_creating-multi-arch-compute-nodes-gcp

Reply:
Excellent!
Added this content as well.